examples/flutter_gallery/lib/demo/material/data_table_demo.dart

```diff
@@ -113,11 +113,11 @@ class DessertDataSource extends DataTableSource {
         }
       },
       cells: <DataCell>[
-        DataCell(Text('${dessert.name}')),
+        DataCell(Text(dessert.name)),
         DataCell(Text('${dessert.calories}')),
```

Comment:
Are some of these needing interpolation? (Other than those appending '%')

Reply:
the other are not String. The string interpolation is used to convert the value to String. They could be replaced by `expression.toString()` (in this case where a string is needed as Text argument). `'${expression}'` vs. `expression.toString()` : not sure I can find a winner.

Reply:
sgtm!

Reply:
FWIW, I would have left this one as-is because now there's weird inconsistencies from line to line.